.travis.yml

```diff
@@ -1,110 +1,42 @@
+language: rust
 sudo: false
 
-language: generic
+cache: cargo
 
-env:
-  global:
-    - PROJECT_NAME=mdBook
+rust:
+  - stable
+  - beta
+  - nightly
 
-matrix:
-  include:
-    # Stable channel
-    - os: osx
-      env: TARGET=i686-apple-darwin CHANNEL=stable
-    - os: linux
-      env: TARGET=i686-unknown-linux-gnu CHANNEL=stable
-      addons:
-        apt:
-          packages: &i686_unknown_linux_gnu
-            - gcc-multilib
-    - os: osx
-      env: TARGET=x86_64-apple-darwin CHANNEL=stable
-    - os: linux
-      env: TARGET=x86_64-unknown-linux-gnu CHANNEL=stable
-      addons:
-        apt:
-          packages:
-            - nodejs
-    - os: linux
-      env: TARGET=x86_64-unknown-linux-musl CHANNEL=stable
-      dist: trusty
-      addons:
-        apt:
-          packages: &musl_packages
-            - musl
-            - musl-dev
-            - musl-tools
-    # Beta channel
-    - os: osx
-      env: TARGET=i686-apple-darwin CHANNEL=beta
-    - os: linux
-      env: TARGET=i686-unknown-linux-gnu CHANNEL=beta
-      addons:
-        apt:
-          packages: *i686_unknown_linux_gnu
-    - os: osx
-      env: TARGET=x86_64-apple-darwin CHANNEL=beta
-    - os: linux
-      env: TARGET=x86_64-unknown-linux-gnu CHANNEL=beta
-    - os: linux
-      env: TARGET=x86_64-unknown-linux-musl CHANNEL=beta
-      dist: trusty
-      addons:
-        apt:
-          packages: &musl_packages
-            - musl
-            - musl-dev
-            - musl-tools
-    # Nightly channel
-    - os: osx
-      env: TARGET=i686-apple-darwin CHANNEL=nightly
-    - os: linux
-      env: TARGET=i686-unknown-linux-gnu CHANNEL=nightly
-      addons:
-        apt:
-          packages: *i686_unknown_linux_gnu
-    - os: osx
-      env: TARGET=x86_64-apple-darwin CHANNEL=nightly
-    - os: linux
-      env: TARGET=x86_64-unknown-linux-gnu CHANNEL=nightly
-    - os: linux
-      env: TARGET=x86_64-unknown-linux-musl CHANNEL=nightly
-      dist: trusty
-      addons:
-        apt:
-          packages: &musl_packages
-            - musl
-            - musl-dev
-            - musl-tools
-
-install:
-  - export PATH="$PATH:$HOME/.cargo/bin"
-  - bash ci/install.sh
+os:
+  - linux
+  - osx
 
 script:
-  - bash ci/script.sh
+  - cargo build --verbose
+  - cargo test --verbose
 
 after_success:
+  # Deploy the docs if the commit is on master
   - test "$TRAVIS_PULL_REQUEST" == "false" &&
     test "$TRAVIS_BRANCH" == "master" &&
-    test "$TARGET" == "x86_64-unknown-linux-gnu" &&
-    test "$CHANNEL" = "stable" &&
+    "$TRAVIS_RUST_VERSION" == "stable" &&
```

Comment:
Shouldn't this line start with `test `? I think it's resulting in some false-positives right now.

Reply:
Ah yes, good catch!